File: app/src/main/java/com/machinelearning/playcarddetect/modules/accessibilityaction/action/OpenApp.java
package com.machinelearning.playcarddetect.modules.accessibilityaction.action;

public class OpenApp extends Action {
    public String packageName ="";
    public OpenApp(int delayTime, String actionType,String packageName) {
        super(delayTime, actionType);
        this.packageName = packageName;
    }

    public String getPackageName() {
        return packageName;
    }

    public void setPackageName(String packageName) {
        this.packageName = packageName;
    }

    public OpenApp() {
    }
}

File: app/src/main/java/com/machinelearning/playcarddetect/modules/accessibilityaction/action/OpenApp.kt
package com.machinelearning.playcarddetect.modules.accessibilityaction.action

import androidx.annotation.NonNull
import androidx.annotation.Nullable
import com.nhatran241.accessibilityactionmodule.model.Action

class OpenApp(@NonNull var packagename: String, @Nullable delaytime : Long) : Action(){
    init {
        this.delayTime = delaytime
    }
}